src/components/Restoration/RestorationType.vue

```diff
@@ -0,0 +1,136 @@
+              <LimitedRestorationPanel :currentDate="getCurrentDate" :expiryDate="getRestoration.expiry"
+                @expiry="setRestorationExpiry($event)" />
+            </v-card>
+          </v-expand-transition>
+
+        </v-radio-group>
+      </v-col>
+    </v-row>
+  </v-card>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { Component, Watch, Emit, Prop } from 'vue-property-decorator'
+import { DateMixin, CommonMixin } from '@/mixins'
+import { Getter, Action } from 'vuex-class'
+import { RestorationTypes, RouteNames } from '@/enums'
+import { RelationshipsPanel } from '@bcrs-shared-components/relationships-panel'
+import { LimitedRestorationPanel } from '@bcrs-shared-components/limited-restoration-panel'
+
+@Component({
+  mixins: [
+    DateMixin,
+    CommonMixin
+  ],
+  components: {
+    RelationshipsPanel,
+    LimitedRestorationPanel
+  }
+})
+export default class RestorationType extends Vue {
+  @Getter getRestoration!: RestorationStateIF
+  @Getter getCurrentDate!: string
+  @Getter isRestorationTypeValid!: boolean
+
+  @Action setRestorationType!: ActionBindingIF
+  @Action setRestorationExpiry!: ActionBindingIF
+  @Action setRestorationRelationships!: ActionBindingIF
+  @Action setRestorationTypeValid!: ActionBindingIF
+
+  // Local properties
+  protected isFull = false
+  protected isLimited = false
+  protected selectRestorationType = ''
+
+  readonly RestorationTypes = RestorationTypes
+
+  /**
+   * Called when component is mounted.
+   * Automatically check limited or full restoration when user continues a draft as per last selection.
+   */
+  mounted (): void {
+    if (this.getRestoration.type === RestorationTypes.LIMITED) {
+      this.selectRestorationType = RestorationTypes.LIMITED
+      this.isFull = !this.isLimited
+    } else {
+      this.selectRestorationType = RestorationTypes.FULL
+    }
+  }
+
+  /**
+   * Set the selected Limited Restoration choice
+   * @param val The value of the selected radio button
+   */
+  @Watch('selectRestorationType')
+  private setLimitedRestorationChoice (val) {
+    this.isLimited = val === RestorationTypes.LIMITED
+    this.isFull = !this.isLimited
+    if (this.isLimited) {
+      this.setRestorationType(RestorationTypes.LIMITED)
+      this.setRestorationRelationships([])
+      this.setRestorationTypeValid(true)
+    } else {
+      this.setRestorationType(RestorationTypes.FULL)
+      this.setRestorationExpiry(null)
+    }
+  }
+
+  @Watch('$route')
+  private async scrollToInvalidComponent (): Promise<void> {
+    if (this.$route.name === RouteNames.RESTORATION_BUSINESS_NAME) {
+      // scroll to invalid components
+      await this.$nextTick()
+      await this.validateAndScroll(
+        {
+          restorationValid: this.isRestorationTypeValid
+        },
+        [
+          'relationships-panel'
+        ]
+      )
+    }
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+@import '@/assets/styles/theme.scss';
+
+</style>
```

Comment:
You don't have to, but you can, delete this style block, since it's not being used.